src/parser/mod.rs

```diff
@@ -388,7 +389,7 @@ impl Parser {
 
                     expect_token!("`=`", self, TokenKind::Equals);
 
-                    let value = self.expression(Precedence::Lowest)?;
```

Comment:
this is a bug fix, declare value cannot be an expression, it needs to be a literal.

Reply:
yup, one of those cases where it was "get something working" :^)